configs/defaults.yaml

```diff
@@ -27,6 +27,10 @@
 #   secret:
 #   session:
 
+# Image Builder configuration
+# ib:
```

Comment:
I think this needs to be `imageBuilder` (it needs to be the same as in the nested config, for viper to pick that up